lib/katello/tasks/upgrades/3.8/clear_checksum_type.rake

```diff
@@ -8,6 +8,10 @@ namespace :katello do
         Katello::RootRepository.yum_type.find_each do |root_repo|
           root_repo.transaction do
             begin
+              if [::Runcible::Models::YumImporter::DOWNLOAD_ON_DEMAND, ::Runcible::Models::YumImporter::DOWNLOAD_BACKGROUND].include?(root_repo.download_policy) && root_repo.url.present? && URI(root_repo.url).scheme == 'file'
+                root_repo.update_attribute(:download_policy, ::Runcible::Models::YumImporter::DOWNLOAD_IMMEDIATE)
+              end
```

Comment:
I dont' know why i didn't realize this before, but updating the katello db isn't enough, it needs update pulp as well.  Updating the importer to be exact, similar to how the distributor is updated below.